emperor/support_files/js/controller.js

```diff
@@ -209,6 +209,9 @@ define([
         controller.resize(w, h);
       }
     });
+
+    // Set for update 
+    this.sceneViews[0].needsUpdate = true;
```

Comment:
Is that always true?

Maybe I missed something but on the design that we did a while ago each scene view was independent so I'm unsure why if 1 needs to change all of them need to.